frontend/projects/upgrade/src/app/core/feature-flags/feature-flags.data.service.ts

```diff
@@ -41,6 +43,30 @@ export class FeatureFlagsDataService {
     return this.http.put<FeatureFlag>(url, flag);
   }
 
+  emailFeatureFlagData(flagId: string, email: string){
+    let featureFlagInfoParams = new HttpParams();
+    featureFlagInfoParams = featureFlagInfoParams.append('experimentId', flagId);
+    featureFlagInfoParams = featureFlagInfoParams.append('email', email);
+
+    const url = this.environment.api.emailFlagData;
+    // return this.http.post(url, { params: featureFlagInfoParams });
+
+    // mock
+    return of(true).pipe(delay(2000));
+  }
+
+  exportFeatureFlagsDesign(flagIds: string[]) {
+    let ids = new HttpParams();
```

Comment:
this will just be one id, so let's remove code for multiple ids. if we want this in the future we can change it then.